packages/components/src/Components/NotAuthorized/NotAuthorized.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon403 from './Icon403';
+import {
+    Title,
+    Button
+} from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+
+const NotAuthorized = ({ serviceName, ...props }) => {
+    let history = useHistory();
+    let button;
+
+    if(history.length) {
+        button = <Button onClick={ history.goBack }>
+            Return to previous page
+        </Button>;
+    } else {
+        button = <Button onClick={ history.push('/') }>
+            Go to landing page
+        </Button>;
+    }
+
+    return (
+        <section {...props} className="pf-l-page__main-section pf-c-page__main-section ins-c-component_not-authorized">
+            <Icon403/>
+            <Title size='3xl'>You do not have access to { serviceName }</Title>
+            <Title headingLevel="h2" size='xl'>Contact your organization administrator(s) for more information.</Title>
+            { button }
+        </section>
+    );
+};
+
+NotAuthorized.propTypes = {
+    serviceName: PropTypes.string
+};
+
+NotAuthorized.defaultProps = {
+    serviceName: undefined
```

Comment:
No need to add defaultProps to `undefined` it is by default `undefined` if you want to have a value in here you could use empty string or null for instance.